RecoTracker/MeasurementDet/plugins/TkPhase2OTMeasurementDet.cc

```diff
@@ -52,6 +36,86 @@ bool TkPhase2OTMeasurementDet::measurements( const TrajectoryStateOnSurface& sta
 
 }
 
+bool TkPhase2OTMeasurementDet::recHits( const TrajectoryStateOnSurface& stateOnThisDet, const MeasurementEstimator& est, const MeasurementTrackerEvent & data,
+                                        RecHitContainer & result, std::vector<float> & diffs) const {
+
+  if unlikely( (!isActive(data)) || isEmpty(data.phase2OTData()) ) return false;
+
+  auto oldSize = result.size();
+
+
+
+  const detset & detSet = data.phase2OTData().detSet(index()); 
+  auto begin = &(data.phase2OTData().handle()->data().front());
+  auto reject = [&](auto ci)-> bool { return (!data.phase2OTClustersToSkip().empty()) && data.phase2OTClustersToSkip()[ci-begin];};
+
+  /// use the usual 5 sigma cut from the Traj to identify the column.... 
+  auto firstCluster = detSet.begin();
+  auto lastCluster = detSet.end();
+
+  // do not use this as it does not account for APE...
+  // auto xyLimits = est.maximalLocalDisplacement(stateOnThisDet,fastGeomDet().specificSurface());
```

Comment:
For documentation. In the old-types (previous century) this method was there to deliver the search limits. Unfortunately does not account for APE.
This code (comments included) has been adapted from IT (read Pixel) MeasurementDet asis in the current realease